pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqldatabase/sqldatabasesismerge/_http.log

```diff
@@ -233,15 +283,20 @@ X-Xss-Protection: 0
       "name": "MYSQL_8_0_39"
     },
     {
-      "displayName": "MySQL 8.0.41",
+      "displayName": "MySQL 8.0.40",
```

Comment:
What happened to 8.0.41? Its on the available list but not on upgradable list.

Reply:
I think it's not in this list bc we are actually using `8.0.40` now!